src/reducers/inputFields.js

```diff
@@ -13,8 +13,30 @@ const initialState = {
   type: "",
   options: [],
   label: "",
-  jsonString: ""
+  jsonString: "",
+  errors: []
 };
+
+const errorMessages = {
+  errorType: "ERROR:Value of Type is incorrect!",
```

Comment:
missing space after `ERROR:` - on the other hand as you always prepend it, you could move `ERROR: ` to input.js when you output it.

Reply:
Okay, I will move it.